README.md

```diff
@@ -124,6 +124,26 @@ Use them with their validator name:
 req.checkBody('users', 'Users must be an array').isArray();
 req.checkQuery('time', 'Time must be an integer great than or equal to 5').isInt().gte(5)
 ```
+####`customSanitizers`
+_{ "sanitizerName": function(value, [additional arguments]), ... }_
+
+The `customSanitizers` option can be used to add additional sanitizers methods as needed. This option should be an `Object` defining the sanitizer names and associated functions.
+
+Define your custom sanitizers:
+
+```javascript
+app.use(expressValidator({
+ customSanitizers: {
+    toStingLowerCase: function(value) {
```

Comment:
I know this is just an example method, but I would prefer for "String" to be spelled correctly. Same for the instance of "toStingLowerCase" below.